src/components/common/Extensions/UserNavigation/partials/UserReputation/UserReputation.tsx

```diff
@@ -0,0 +1,134 @@
+import clsx from 'clsx';
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import {
```

Comment:
Add an empty line between imports packages and other imports (like hooks, types, components, etc)